cpp/tests/c_api/mg_test_utils.cpp

```diff
@@ -191,3 +191,102 @@ extern "C" int create_mg_test_graph(const cugraph_resource_handle_t* handle,
 
   return test_ret_value;
 }
+extern "C" int create_mg_test_graph_with_ids(const cugraph_resource_handle_t* handle,
```

Comment:
`with_edge_ids` might be more intuitive?

Reply:
Fixed in latest push